user/id.go

```diff
@@ -71,6 +71,9 @@ func (x *ID) SetScriptHash(scriptHash util.Uint160) {
 //
 // Return value MUST NOT be mutated: to do this, first make a copy.
```

Comment:
And here.

Reply:
Updated

Reply:
Looks like this one is still there.

Reply:
Fixed this and one more place. Hope it was the last one. Also rebased